src/test/regress/citus_tests/run_test.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pathlib
+from glob import glob
+import argparse
+import shutil
+import random
+import re
+import common
+
+args = argparse.ArgumentParser()
+args.add_argument("-t", "--test", required=True, help="Relative path for test file (must have a .sql or .spec extension)", type=pathlib.Path)
```

Comment:
Can you verify that the `test-name` parameter is a file name in sql/spec folder in `src/test/regress`? So I prefer using a relative path instead of a `test-name` to avoid searching for it in the folders, handling conflicts, etc. in `run-test.py`.

Reply:
I also don't think you have to search for it in the directory, since you're already searching for it in the schedules.